web-app/pages/api/cert/[imageFileName].ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines-per-function */
```

Comment:
Please don't disable max-lines-per-function up here at the top, where it would apply to the whole file. And let's ideally not disable it anywhere. But *occasionally* there might be one function that needs ~32 lines or so and doesn't seem like it needs to be split into smaller functions. But that might be rare.

Reply:
I couldn't make `fetchCertificateDetails` less than 30 lines so I put `// eslint-disable-next-line max-lines-per-function` just above the `fetchCertificateDetails `